Automattic/sensei
# Add course list pattern with background for grid and list style

Implements https://github.com/Automattic/sensei/issues/6050

### Changes proposed in this Pull Request

* Added pattern markup and styles for with course list and wide background

### Testing instructions

- Checkout this branch
- Checkout the trunk of [Course](https://github.com/Automattic/course)
- Create a new page. If you create a post or course, make sure to have a wide (1000px for the Course Theme) template
- In the editor, go to the inserter, select Sensei LMS from the dropdown
- Make sure Course List and Course List - Grid patterns are there
- Add these patterns and make sure they match the design in the frontend

### Screenshot / Video

Grid
<img width="1933" alt="Screen Shot 2022-11-24 at 10 45 31 AM" src="https://user-images.githubusercontent.com/6820724/203696516-ad17a3bd-6e33-4645-b62a-36d12b081824.png">

List

![Screen Shot 2022-11-24 at 10 46 09 AM](https://user-images.githubusercontent.com/6820724/203696534-f56dac68-1d11-4834-8dbc-6427a08c0c8c.png)
